README.md

```diff
@@ -149,9 +149,9 @@ These methods are available on a Writer object:
   Connect to the nsqd specified.
 * `close()` <br/>
   Disconnect from nsqd.
-* `publish(topic, msgs, [callback])` <br/>
+* `publish(topic, msgs, [timeMs], [callback])` <br/>
   `topic` is a string. `msgs` is either a string, a `Buffer`, JSON serializable
-  object, a list of strings / `Buffers` / JSON serializable objects. `callback` takes a single `error` argument.
+  object, a list of strings / `Buffers` / JSON serializable objects. `timeMs` is the delay by which the message should be delivered. `callback` takes a single `error` argument.
```

Comment:
As I see it as of now. There is no way to get the the max-req-timeout from the server. But it doesn't matter. Because there is clamping going on. Which means it will still delay it but with maxium req timeout. https://github.com/nsqio/nsq/pull/868